c/datatablemodule.cc

```diff
@@ -101,6 +102,7 @@ PyObject* register_function(PyObject*, PyObject *args) {
   else if (n == 6) replace_dtWarning(fnref);
   else if (n == 7) py::Frame_Type = fnref;
   else if (n == 8) py::fread_fn = fnref;
+  else if (n == 9) py::fallback_makedatatable = fnref;
```

Comment:
Any reason not to use `switch` here?

Reply:
it grew over time :) But really, this code is executed only once, during import, so not a big deal